app/models/food.rb

```diff
@@ -1,6 +1,27 @@
 class Food < ApplicationRecord
   belongs_to :category
   has_one_attached :image
+  scope :search, lambda {|term|
+    return if term.blank?
+
+    where("foods.name LIKE :term OR foods.description LIKE :term", term:
+       "%#{term}%")
+  }
+  scope :all_of_category, ->(category_id){where category_id:}
+  scope :filter_by_price, lambda {|from, to|
+    return if from.blank? && to.blank?
+
+    if to.blank?
+      where("price >= ?", from)
+    elsif from.blank?
+      where("price <= ?", to)
+    else
+      where("price >= ? AND price <= ?", from, to)
+    end
```

Comment:
```suggestion
      where("price >= ?", from).where("price <= ?", to)
```
Như thế này thôi thì sao nhỉ

Reply:
Dạ vâng ạ